packages/moonstone/internal/$L/$L.js

```diff
@@ -32,8 +32,10 @@ function createResBundle (options) {
 	if (typeof ILIB_MOONSTONE_PATH !== 'undefined') {
 		opts = {
 			loadParams: {
+				// Deprecated; to be removed in future
 				root: ILIB_MOONSTONE_PATH
```

Comment:
Why are we retaining this if it's deprecated?

Reply:
Backward compatibility with iLib 14.0.0 through 14.3.0.